playbooks/dataiku.yml

```diff
@@ -1,40 +1,38 @@
 - hosts: 127.0.0.1
   connection: local
-  become_user: ubuntu
   gather_facts: no
-  roles:
-    - python3.6-base
   vars_files:
     - ./group_vars/dataiku
 
   tasks:
-    - name: I am confused b
-      command: 'whoami'
-      register: myidentity
-
-    - name: my secret identity
-      debug:
-        msg: '{{ myidentity.stdout }}'
-
-    - name: install default-jdk
-      apt:
-        name: openjdk-11-jdk
+    - name: Increase system limits as required by DSS
+      become: true
+      pam_limits:
+        domain: "{{dss_service_user}}"
+        limit_item: "{{item}}"
+        limit_type: "-"
+        value: 65536
+      loop:
+        - nofile
+        - nproc
+   
+    - name: Update apt-get repo and cache
+      become: true
+      apt: update_cache=yes force_apt_get=yes cache_valid_time=3600
+
+    - name: Create service user
+      become: true
```

Comment:
Is `become` needed? AFAIK RSC deals with this nicely, or is it just best practice to create users as root instead of another user?